ArduPlane/quadplane.cpp

```diff
@@ -1116,6 +1116,16 @@ void QuadPlane::control_qacro(void)
     }
 }
 
+void QuadPlane::relax_attitude_control(){
```

Comment:
please follow style guide

Reply:
fixed
this must have been an accidental newline deletion, since if I had done it on purpose it would have been ") {" 
:)